src/components/shadows/examples/button-shadows.tsx

```diff
@@ -0,0 +1,76 @@
+import { Component, h, State } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-button-shadows',
+    shadow: true,
+    styleUrl: 'button-shadows.scss',
+})
+export class SizeExample {
+    @State()
+    private toggleDarkLightModes: boolean = false;
+
+    constructor() {
+        this.toggleMode = this.toggleMode.bind(this);
+    }
+    public render() {
+        return [
+            <div
+                id="button-shadow-example"
+                class={{ 'visualize-sizes': this.toggleDarkLightModes }}
+            >
+                <div class="visualization">
+                    <button class="button normal">
+                        <div class="label one">Normal</div>
+                    </button>
+                    <code>--button-shadow-normal</code>
+                    <p>makes the element look clickable</p>
+                </div>
+                <div class="visualization">
+                    <button class="button hovered">
+                        <div class="label two">Hover</div>
+                    </button>
+                    <code>--button-shadow-hover</code>
+                    <p>makes the element lifted up, ready to be pressed</p>
+                </div>
+                <div class="visualization">
+                    <button class="button pressed">
+                        <div class="label three">Pressed</div>
+                    </button>
+                    <code>--button-shadow-pressed</code>
+                    <p>
+                        renders the element pressed down towards the surfase
+                        below
+                    </p>
+                </div>
+                <div class="visualization">
+                    <button class="button inset">
+                        <div class="label three">Inset</div>
+                    </button>
+                    <code>--button-shadow-inset</code>
+                    <p>
+                        renders the element pressed down lower than its
+                        background surfase
+                    </p>
+                </div>
+                <div class="visualization">
+                    <button class="button inset-pressed">
+                        <div class="label three">Inset</div>
+                    </button>
+                    <code>--button-shadow-inset-pressed</code>
+                    <p>makes the inset elements look even more pressed down</p>
+                </div>
+            </div>,
+            <limel-flex-container justify="start">
+                <limel-checkbox
+                    label="Dark/light mode"
```

Comment:
This isn't a good label for the same reason `toggleDarkLightModes` isn't a good name for a boolean variable: it's not clear what one should expect from a checked or unchecked box.

Reply:
this is removed